src/App.tsx

```diff
@@ -26,70 +27,130 @@ type ReorderOptions = {
   isReversed: boolean,
 };
 
-// Use this function in the render to prepare goods
 export function getReorderedGoods(
   goods: string[],
   { sortType, isReversed }: ReorderOptions,
 ) {
-  // To avoid the original array mutation
   const visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
-  // eslint-disable-next-line no-console
-  console.log(sortType, isReversed);
+  visibleGoods.sort((item1, item2) => {
+    switch (sortType) {
+      case 1:
+        return item1.localeCompare(item2);
+
+      case 2:
```

Comment:
I think it's better to name cases according to the action they perform

Reply:
good point, but i thought that will be more... accuracy)
but if mentors will say remake, i will) thx)